ivy/functional/frontends/torch/blas_and_lapack_ops.py

```diff
@@ -209,6 +210,22 @@ def trapezoid(y, x=None, *, dx=None, dim=-1):
     return ivy.trapz(y, x=x, dx=dx, axis=dim)
 
 
+@ivy.to_ivy_arrays_and_back
+def triangular_solve(A, b, upper=False, transpose=False, unitriangular=False):
+    if upper:
+        A = ivy.transpose(A, (-2, -1)) if transpose else A
+        "T" if transpose else "N"
```

Comment:
What is going on here

Reply:
There was a typographical error, which now has been corrected.